src/core/InterleavedBufferAttribute.js

```diff
@@ -5,27 +5,18 @@ const _vector = new /*@__PURE__*/ Vector3();
 
 class InterleavedBufferAttribute {
 
-	constructor( interleavedBuffer, itemSize, offset, normalized ) {
+	constructor( buffer, itemSize, type, normalized, stride, offset, count ) {
```

Comment:
I think that I would swap the order of `type` and `normalized`, to match existing BufferAtribute constructors. In this case `normalized` would be a required parameter; seems fine to me.

Reply:
The number of parameters will be many so I thought making the order be same as `gl.vertexAttribPointer()` parameters help users understand the order, but no strong opinion. Swapping them sounds ok to me.